### PR TITLE
PaaS - MEP Service Permissions

### PR DESCRIPTION
**Pull request by dilallkx**
**CHANGES:**
- Permissions Configuration:
  - Permissions endpoint updates to match latest code base
  - New 'api' config point to support services with multiple APIs
  - File cleanup
- Auth Service:
  - Permissions verification for services with multiple APIs
  - Permissions verification for MEP services
    - NOTE: Currently supports one permissions config point for all Sandbox/MEP deployments of a single service (i.e. no sandbox or mep-specific permissions)

**TESTING:**
- Manual verification of permissions rules

### CODE REVIEW

File: go-apps/meep-auth-svc/server/auth-svc.go [hunk 1]
```diff
@@ -365,32 +367,58 @@ func cacheServicePermissions(cfg *PermissionsConfig) {
...
+				// Sandbox service endpoint
+				route = new(AuthRoute)
+				route.Prefix = false
+				route.Method = ep.Method
+				route.Name = apiPrefix + ep.Name
 				route.Pattern = "/{sbox}" + svc.Path + ep.Path
+				routes = append(routes, route)
+				svcMap[route.Name] = permission
 			} else {
+				// Global service endpoint
```
Comment (pastorsx) [inline go-apps/meep-auth-svc/server/auth-svc.go:413]: lot of common code in the if/else... could be taken out of the if/else

File: go-apps/meep-auth-svc/server/auth-svc.go [hunk 2]
```diff
@@ -407,18 +435,33 @@ func cacheServicePermissions(cfg *PermissionsConfig) {
...
+
+			// Sandbox service
+			route = new(AuthRoute)
+			route.Prefix = true
+			route.Name = apiPrefix + svc.Name
 			route.Pattern = "/{sbox}" + svc.Path
+			routes = append(routes, route)
+			svcMap[route.Name] = permission
 		} else {
+			// Global service
```
Comment (pastorsx) [inline go-apps/meep-auth-svc/server/auth-svc.go:457]: same as previous comment

File: go-apps/meep-auth-svc/server/auth-svc.go [hunk 3]
```diff
@@ -432,25 +475,40 @@ func cacheFileserverPermissions(cfg *PermissionsConfig) {
...
+
+			// Sandbox fileserver
+			route = new(AuthRoute)
+			route.Prefix = true
+			route.Name = fs.Name
 			route.Pattern = "/{sbox}" + fs.Path
+			routes = append(routes, route)
+			authSvc.cache.Fileservers[route.Name] = permission
 		} else {
+			// Global fileserver
```
Comment (pastorsx) [inline go-apps/meep-auth-svc/server/auth-svc.go:504]: some common code could be taken out of if/else

**Review by pastorsx — APPROVED** [review]
small comments that are just suggestions... approved

File: go-apps/meep-auth-svc/server/auth-svc.go (hunk 1 above)
Comment (dilallkx) [reply go-apps/meep-auth-svc/server/auth-svc.go:413]: Agreed. Will try to address when working in this module again.

File: go-apps/meep-auth-svc/server/auth-svc.go (hunk 2 above)
Comment (dilallkx) [reply go-apps/meep-auth-svc/server/auth-svc.go:457]: Agreed. Will try to address when working in this module again.

File: go-apps/meep-auth-svc/server/auth-svc.go (hunk 3 above)
Comment (dilallkx) [reply go-apps/meep-auth-svc/server/auth-svc.go:504]: Agreed. Will try to address when working in this module again.

**Pull request merged**